app/controllers/employers/plan_years_controller.rb

```diff
@@ -39,8 +39,7 @@ def reference_plans
       Plan.by_active_year(params[:start_on]).for_service_areas(@employer_profile.service_area_ids).shop_market.health_coverage.by_carrier_profile(@carrier_profile).and(hios_id: /-01/)
     elsif params[:plan_option_kind] == "metal_level"
       @metal_level = params[:metal_level]
-      @available_carrier_ids = CarrierProfile.carriers_for(@employer_profile)
-      Plan.by_active_year(params[:start_on]).for_service_areas_and_carriers(@employer_profile.service_area_ids, @available_carrier_ids).shop_market.health_coverage.by_metal_level(@metal_level).and(hios_id: /-01/)
+      Plan.by_active_year(params[:start_on]).for_service_areas_and_carriers(CarrierProfile.carrier_profile_service_area_pairs_for(@employer_profile), TimeKeeper.date_of_record.year).shop_market.health_coverage.by_metal_level(@metal_level).and(hios_id: /-01/)
```

Comment:
Minor nitpick - shouldn't the active year value be the same for both by_active_year and for_service_areas_and_carriers?  Consider an employer who is setting up a plan year in november.